enterprise/frontend/src/embedding-sdk-ee/tests/use-init-data/setup.tsx

```diff
@@ -29,9 +27,9 @@ import type {
   MetabaseAuthConfig,
 } from "embedding-sdk-bundle/types";
 
-import { useInitDataInternal } from "../use-init-data-internal";
+import { setupEmbeddingSdkEnterprisePlugins } from "../support";
 
-jest.mock("embedding/auth-common/saml-token-storage", () => {
+jest.mock("metabase/embedding-sdk/lib/saml-token-storage", () => {
```

Comment:
I originally put it in `auth-common` there to make sure it can be safely put in `embed.js` without being polluted by dependencies (i.e. someone accidentally importing stuff in and bloating embed.js). At the very least could we put it in a `metabase/embedding-sdk/auth` module?